ding/policy/mbpolicy/mbsac.py

```diff
@@ -219,3 +221,197 @@ def _monitor_vars_learn(self) -> List[str]:
             'alpha',
             'target_q_value',
         ] + alpha_loss
+
+
+@POLICY_REGISTRY.register('stevesac')
+class STEVESACPolicy(SACPolicy):
```

Comment:
Origin STEVE uses DDPG as policy improvement, so we should replace the sac?

Reply:
STEVE uses DDPG because its baseline algorithm MVE uses DDPG, which is published before the SAC. Since STEVE is no longer SOTA, a complete reimplementation is not necessary. But the idea of Stochastic Ensemble Value Expansion is worth integrating into our model-based toolbox.